Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:6
+FROM node:10

```

Comment:
This is actually a huge image.  We should be able to switch to `node:10-alpine`, but that will require a couple of other tweaks.